tools/codegen/dest/register_dispatch_key.py

```diff
@@ -129,9 +178,19 @@ def gen_unstructured(self, f: NativeFunction) -> Optional[str]:
             return result
         elif self.target is Target.NAMESPACED_DEFINITION:
             def generate_defn(cpp_sig: CppSignature) -> str:
+                # This is needed in order for namespaced definitions to call into CPU fallbacks,
+                # which live in a different namespace.
+                # TODO: this logic will change if we implement a boxed CPU fallback.
+                if isinstance(native_or_external, ExternalBackendFunction) \
```

Comment:
if we want `at::xla::{op}` to work for all ops (including cpu fallbacks), this is where the extra logic lives.

Reply:
I take it back, maybe we shouldn't generate `at::xla` 😂